src/Plugin/EncryptionMethod/McryptAES256Encryption.php

```diff
@@ -12,7 +12,8 @@
  * @EncryptionMethod(
  *   id = "mcrypt_aes_256",
  *   title = @Translation("Mcrypt AES 256"),
- *   description = "This uses PHPs mcrypt extension and <a href='http://en.wikipedia.org/wiki/Advanced_Encryption_Standard'>AES-256</a>."
+ *   description = "This uses PHPs mcrypt extension and <a href='http://en.wikipedia.org/wiki/Advanced_Encryption_Standard'>AES-256</a>.",
```

Comment:
This class can be deleted, we don't need it.

We have encrypt_test now, which was my lone concern about having a relevant test encryption method to ship encypt with